backend/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:14-slim AS dev
+RUN mkdir -p /backend
+WORKDIR /backend
```

Comment:
No need sub directory inside a docker container. It should be place on the `root`

Reply:
Fixed